public/js/grid/react_util.js

```diff
@@ -0,0 +1,115 @@
+var CoursePanel = React.createClass({
+    render: function() {
+        return (
+            <ul className="trapScroll-enabled" id="course-select">
+                <li id="clear-all">
+                    <h3>Clear All</h3>
+                </li>
+            </ul>
+        );
+    }
+});
+
+var SearchPanel = React.createClass({
+    render: function() {
+        return (
+            <div>
+                <div id="filter-container">
+                    <form onsubmit="return false;">
```

Comment:
@david-yz-liu there's some CSS tied to the "form" element, so removing it would require some modification to existing CSS. Should I do that?